.eslintrc.json

```diff
@@ -1,15 +1,45 @@
 {
+  "env": {
+    "jest": true
+  },
   "extends": "airbnb",
-  "globals": {   
+  "globals": {
     "window": true,
     "browser": true,
-    "node": true
+    "node": true,
+    "document": true,
+    "FileReader": true
   },
   "rules": {
-    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-    "indent": ["error", 4],
-    "react/jsx-indent": ["error", 4],
-    "react/jsx-indent-props": ["error", 4],
+    "react/jsx-props-no-spreading": [
+      2,
+      {
+        "html": "enforce",
+        "custom": "ignore",
+        "exceptions": []
+      }
+    ],
+    "react/jsx-filename-extension": [
+      1,
+      {
+        "extensions": [
+          ".js",
+          ".jsx"
+        ]
+      }
+    ],
+    "indent": [
+      "error",
+      4
+    ],
+    "react/jsx-indent": [
+      "error",
+      4
+    ],
+    "react/jsx-indent-props": [
+      "error",
+      4
+    ],
     "import/extensions": "off",
     "import/no-unresolved": "off",
     "class-methods-use-this": 0,
```

Comment:
Why disabled class-methods-use-this? 
Maybe creating static methods looks better.
https://eslint.org/docs/rules/class-methods-use-this

Reply:
Totally agreed. I'll remove it.